src/app/tab2/tab2.page.html

```diff
@@ -13,28 +11,22 @@
     </ion-toolbar>
   </ion-header>
 
-  <app-explore-container name="Tab 2 page"></app-explore-container>
```

Comment:
Was this just unnecessary and did it add to the error?

Reply:
Well the explore container is another component that you where invoking, basically you where calling that page to be inside there, but you didn't have any reason to implement that component